PROTO_tests/tests/operator_test.py

```diff
@@ -549,9 +551,11 @@ def test_str_repr(self):
         assert ak.array([1.1, 2.3, 5]).__repr__() in answers
 
         answers = [
-            "array([0 0.52631578947368418 1.0526315789473684 ... 8.9473684210526319 9.473684210526315 10])",
+            "array([0 0.52631578947368418 1.0526315789473684 ... 8.9473684210526319 "
+            "9.473684210526315 10])",
             "array([0 0.5 1.1 ... 8.9 9.5 10])",
-            "array([0.00000000000000000 0.52631578947368418 1.0526315789473684 ... 8.9473684210526319 9.473684210526315 10.00000000000000000])",
+            "array([0.00000000000000000 0.52631578947368418 1.0526315789473684 ... 8.9473684210526319 "
+            "9.473684210526315 10.00000000000000000])",
```

Comment:
sorry I know i already approved this but I just notice this never got put back to the less confusing format

Reply:
Whoops, good catch.